Automattic/jetpack
# Launchpad: Add Manage subscribers task for post-launch newsletter task list

Fixes https://github.com/Automattic/wp-calypso/issues/79793

## Proposed changes:

* Add a "Manage your subscribers" task to the Newsletter post-launch task lists.
* Only show the task if the user imported subscribers during onboarding; this means they have the `import-subscribers` `site_goal` option.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply this patch and sandbox `public-api.wordpress.com`
* Create a new site from `/setup/newsletter` and select the import subscribers intent during onboarding.
* Go the developer console and GET from WP REST API `wpcom/v2/sites/siteSlug/launchpad?checklist_slug=intent-free-newsletter` and `wpcom/v2/sites/siteSlug/launchpad?checklist_slug=intent-paid-newsletter`
* The `manage_subscribers` task should show in the response:

<img width="1151" alt="Screenshot 2023-07-25 at 12 42 11 PM" src="https://github.com/Automattic/jetpack/assets/2124984/6693462a-2a46-4250-82cd-bc58ed528e20">

* To further confirm the visibility is working, create a new site with the Free newsletter intent from `/setup/newsletter`
* Go the developer console and GET from WP REST API `wpcom/v2/sites/siteSlug2/launchpad?checklist_slug=intent-free-newsletter` and `wpcom/v2/sites/siteSlug2/launchpad?checklist_slug=intent-paid-newsletter`
* The `manage_subscribers` task should *not* show in the response.
